<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>986</width>
    <height>773</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>988</width>
    <height>773</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#top{
background-color: qlineargradient(x1:1,y1:0,x2:0,y2:0,stop:0  #29bdd9,stop:1 #276ace);
}</string>
     </property>
     <widget class="QPushButton" name="user_name">
      <property name="geometry">
       <rect>
        <x>414</x>
        <y>7</y>
        <width>151</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border:none;
color:white;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="min_btn">
      <property name="geometry">
       <rect>
        <x>888</x>
        <y>2</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="close_btn">
      <property name="geometry">
       <rect>
        <x>938</x>
        <y>2</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mid" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:white;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="message_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView{
border:none;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>182</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>182</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottom{
background-color:white;
border-top:1px solid grey;
}</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>988</width>
        <height>46</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>46</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>46</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="bottom_close">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>130</y>
        <width>92</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>92</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>92</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="message">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>46</y>
        <width>988</width>
        <height>84</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>84</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>84</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#message{
border:none;
}</string>
      </property>
     </widget>
     <widget class="QtMaterialRaisedButton" name="send">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>98</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>92</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
